python/sglang/srt/server.py

```diff
@@ -434,6 +434,8 @@ def _set_envs_and_config(server_args: ServerArgs):
             "at https://docs.flashinfer.ai/installation.html.",
         )
 
+    mp.set_start_method("spawn", force=True)
```

Comment:
only do this for hip for now. We will figure out a better method of not using fork later.